Userland/Libraries/LibVideo/VP9/BitStream.cpp

```diff
@@ -167,9 +167,9 @@ ErrorOr<u8> BitStream::read_literal(u8 bit_count)
     return result;
 }
 
-ErrorOr<i8> BitStream::read_s(size_t bits)
+ErrorOr<i16> BitStream::read_s(size_t bits)
 {
-    auto value = TRY(read_bits(bits));
+    i16 value = TRY(read_bits(bits));
```

Comment:
This is change is not explained in the commit message, and could belong in a separate commit.

Reply:
It looks like this change may not have been necessary after all, so I'll revert it for now.